tests/MongoDB.Driver.Core.Tests/Core/Operations/EndTransactionOperationTests.cs

```diff
@@ -93,17 +96,33 @@ public void CommandName_should_return_expected_result()
             result.Should().BeSameAs("fakeOperation");
         }
 
+        [Fact]
+        public void RecoveryToken_should_return_expected_result()
```

Comment:
Wrong test ordering?

Reply:
I'm not sure what the ordering is for this file, but it's definitely not alpha-sorted

Reply:
yep, but maybe it's better to add this test to the end of the file with the assumption that last `CommandName..` and `CreteCommand..` tests are placed in the right order..?

Reply:
Sure, moved!

Reply:
I use a slightly different ordering in test files than in production files:

* all the public test methods first, in the same order as the members they are testing in the production file (makes it easy to verify that all members are being tested)
* all private helper methods at the end

Reply:
Thanks for the clarification Robert!